abci/types/messages.go

```diff
@@ -226,3 +242,19 @@ func ToResponseFinalizeBlock(res *ResponseFinalizeBlock) *Response {
 		Value: &Response_FinalizeBlock{res},
 	}
 }
+
+//
+// side channel
```

Comment:
If we keep the comments, can we add the following?
`// TODO CHECK HEIMDALL-V2 check implementation of SideChannel`

Reply:
Not needed no more.